lib/msf/core/post/windows/extapi.rb

```diff
@@ -6,21 +6,6 @@ module Windows
 
 module ExtAPI
 
-  def initialize(info = {})
-    super(
-      update_info(
-        info,
-        'Compat' => {
-          'Meterpreter' => {
-            'Commands' => %w[
-              extapi_*
-            ]
-          }
-        }
-      )
-    )
-  end
```

Comment:
modules now uses the more granular extapi_* metadata, rather than `extapi` requiring all commands to be implemented